internal/manifest/vpc.go

```diff
@@ -35,8 +37,13 @@ func NewVPC(m *Manifest,
 func (p *VPC) serialize() osbuild.Pipeline {
 	pipeline := p.Base.serialize()
 
+	formatOptions := osbuild.VPCOptions{
+		Type:      "vpc",
```

Comment:
nitpick: this can be omitted, because `NewQEMUStageOptions` will set it if not set based on the type of the specific `formatOptions` type. But I'm OK with keeping it as is, but in that case you should use `osbuild.QEMUFormatVPC` instead of a string literal.

Reply:
I think omitting and letting it get set automatically is better.  I'll fix it.